tests/Predis/ClientTest.php

```diff
@@ -1048,7 +1048,7 @@ public function testPubSubLoopWithoutArgumentsReturnsPubSubConsumer(): void
     {
         $client = new Client();
 
-        $this->assertInstanceOf('Predis\PubSub\Consumer', $client->pubSubLoop());
+        $this->assertInstanceOf('Predis\Consumer\PubSub\Consumer', $client->pubSubLoop());
```

Comment:
TBH I preferred the old pathing. It looks prettier. Any way to approach it?

Reply:
@chayim It was done to keep different entities within the same domain. There are a lot of "Consumers": PubSub, Sharded PubSub, Push Notifications. All of them "consume" messages from server. IMHO, it's better to create a separate place where we can keep all entities related to this domain, then spreading them across whole library.